apps/core/views.py

```diff
@@ -234,10 +236,10 @@ def get(self, request, *args, **kwargs):
         return Response({"reset_token": reset_token})
 
 
-class PasswordChangeView(APIView):
+class PasswordResetChangeView(APIView):
```

Comment:
Why we need PasswordResetChangeView and PasswordChangeView
What is the difference between them? PasswordResetChangeView seems to cover multiple cases or not?

Reply:
Then remove the Change particle from the name of PasswordResetChangeView if this component is used only if 
> when password is forgotten and user isn't authenticated

Reply:
Sorry, I really can't get it. 
Let's discuss this online on Saturday.

Reply:
Add comments into view, where they used